plugins/linear-genome-view/src/LinearGenomeView/components/TrackLabel.tsx

```diff
@@ -103,7 +103,7 @@ const TrackLabel = React.forwardRef(
     }
 
     const items = [
-      ...session.getTrackActionMenuItems?.(trackConf),
+      ...(session.getTrackActionMenuItems?.(trackConf) || []),
```

Comment:
new lint warned this was actually illegal if spread undefined, which is true! "undefined is not iterable"